app/src/examples/VolumeExample.tsx

```diff
@@ -0,0 +1,211 @@
+import Animated, {
+  useSharedValue,
+  useAnimatedProps,
+  useAnimatedSensor,
+  SensorType,
+  useFrameCallback,
+  useAnimatedStyle,
+} from 'react-native-reanimated';
+import {
+  View,
+  StyleSheet,
+  Text,
+  Switch,
+  TextInput,
+  StatusBar,
+} from 'react-native';
+import React, { useLayoutEffect } from 'react';
+import { useNavigation } from '@react-navigation/native';
+import Slider from '@react-native-community/slider';
+
+const primaryColor = '#FFD61E';
+const secondaryColor = '#f5f5f5';
+const backgroundColor = '#232736';
+const secondaryBackgroundColor = '#1B2445';
+
+const AnimatedTextInput = Animated.createAnimatedComponent(TextInput);
+Animated.addWhitelistedNativeProps({ text: true });
+
+const AnimatedSlider = Animated.createAnimatedComponent(Slider);
+Animated.addWhitelistedNativeProps({ value: true });
+
+export default function VolumeExample() {
+  const navigation = useNavigation();
+
+  useLayoutEffect(() => {
+    navigation.setOptions({
+      headerLargeTitle: true,
+      headerTransparent: false,
+      headerTitle: 'Volume',
+      headerStyle: {
+        backgroundColor,
+      },
+      headerLargeTitleStyle: {
+        color: '#f7f7ff',
+        fontFamily: 'Poppins-Bold',
+      },
+    });
+  }, [navigation]);
+
+  const sv = useSharedValue(50);
+
+  const x = useSharedValue(50);
+  const vx = useSharedValue(0);
+
+  const animatedSensor = useAnimatedSensor(SensorType.GRAVITY);
+
+  useFrameCallback(({ timeSincePreviousFrame: dt }) => {
+    if (dt == null) {
+      return;
+    }
+    const ax = animatedSensor.sensor.value.x * 0.0001;
+    vx.value += ax * dt;
+    x.value += vx.value * dt;
+    x.value = Math.min(100, Math.max(0, x.value));
+
+    if (x.value === 0 || x.value === 100) {
+      vx.value = 0;
+    }
+  });
```

Comment:
Just wondering 🤔 Is a frame callback necessary here? Maybe the better idea would be the usage of the `useDerivedValue` or `useAnimationReaction` hook here.

Reply:
Hm 🤔 I kinda need the change in time `dt` for physics calculations